xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-theme/xwiki-platform-flamingo-theme-ui/src/main/resources/FlamingoThemesCode/ThemeSheet.xml

```diff
@@ -322,6 +322,10 @@
 &lt;div class="row"&gt;
   #if($isEdit)
     #displayVariablesPanel()
+    ## Always hide both side panel columns
+    #set($showLeftPanels = "0")
+    #set($showRightPanels = "0")
+    &lt;script&gt;document.getElementById("body").classList.add("hidelefthideright");&lt;/script&gt;
```

Comment:
I don't like the inline script. If the problem is that ``$showLeftPanels`` and ``$showRightPanels`` are set too late, then maybe we can set them in a UI extension that is executed before they are used (checking before that we are in edit mode and inside the ``FlamingoThemesCode`` space, of course).